### DISCUSSION TITLE
Aggregation query in morphia 2.2.3 version

### DISCUSSION THREAD

**Discussion by Shalaka1197** [MorphiaOrg/morphia#2386] [General]
I am writing the query for aggregate function using morphia 2.2.3 version. The query code is deprecated when I use below query.

Morphia query - mongoDatastore.createAggregation(Classname).group().sort().aggregate(Classname);

Please let me know if there is any alternative available in this version for the same query.
 
 - Server Version:4.4.0
 - Driver Version:4.4.0
 - Morphia Version:2.2.3

**Comment by evanchooly** [comment]
use `Datastore#aggregate(Class)` instead.
